autoscheduler/frontend/src/components/LandingPage/LandingPage.tsx

```diff
@@ -1,15 +1,22 @@
 import * as React from 'react';
 import { RouteComponentProps } from '@reach/router';
-
+import { useDispatch } from 'react-redux';
 import HelpText from './HelpText/HelpText';
 import SelectTerm from './SelectTerm/SelectTerm';
 import * as styles from './LandingPage.css';
+import setTerm from '../../redux/actions/term';
+
+const LandingPage: React.FC<RouteComponentProps> = () => {
+  // Set the term to null so we don't display a term in the navbar when we're on this page
+  const dispatch = useDispatch();
+  dispatch(setTerm(null));
```

Comment:
It's not a big deal since it's functionally the same, but this would be better off as a `useEffect` with no dependencies since the intended behavior is to set the term to null whenever the user goes to the landing page (not every time it rerenders)